src/components/ActiveFilters.vue

```diff
@@ -0,0 +1,109 @@
+<template>
+  <div>
+    <button
+      v-for="(item, key) in activeValues"
+      :key="key"
+      type="button"
+      class="active-filter btn btn-light m-1 btn-outline-secondary"
+      @click="removeFilter(item)"
+    >
+      {{ item.label }}: {{ item.value }}
+      <font-awesome-icon
+        icon="times"
+        class="ml-1"
+      />
+    </button>
+  </div>
+</template>
+
+<script>
+import Vue from 'vue'
+
+import { library } from '@fortawesome/fontawesome-svg-core'
+import { faTimes } from '@fortawesome/free-solid-svg-icons'
+import { FontAwesomeIcon } from '@fortawesome/vue-fontawesome'
+library.add(faTimes)
+
+export default Vue.extend({
+  name: 'ActiveFilters',
+  components: { FontAwesomeIcon },
+  props: {
+    filters: {
+      type: Array,
+      required: true
+    },
+    value: {
+      type: Object,
+      default: () => ({})
+    }
+  },
+  data () {
+    return {
+      isOpen: this.collapsable ? !this.collapsed : true
+    }
+  },
+  computed: {
+    activeValues () {
+      let values = []
+      Object.keys(this.value).map(key => {
+        const current = this.value[key]
```

Comment:
could use entries ( key and value)
 https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/entries